packages/psss-server/src/connections/ApiConnection.ts

```diff
@@ -23,7 +23,7 @@ export class ApiConnection {
     return this.request('GET', endpoint, queryParameters);
   }
 
-  post(endpoint: string, queryParameters: QueryParameters, body: RequestBody) {
+  post(endpoint: string, queryParameters: QueryParameters, body?: RequestBody) {
```

Comment:
Making this optional because fetchingReport is a post but only requires body when sending test data

Reply:
I'm not sure it should be a post then?